frontend/pages/_document.jsx

```diff
@@ -6,12 +6,9 @@ class MyDocument extends Document {
       <Html lang="en-US" style={{ fontSize: '14px' }}>
         <Head>
           <script
+            rel="preconnect"
```

Comment:
Preconnect and dns-prefetch should be used on the domains 
like: 
<link rel="preconnect" href="https://www.google-analytics.com" />
          <link rel="dns-prefetch" href="https://www.google-analytics.com" />
          <link rel="preconnect" href="https://www.googletagmanager.com" />
          <link rel="dns-prefetch" href="https://www.googletagmanager.com" />

Reply:
I think we can not add the `rel` attribute in `iframe`
`<iframe src="https://www.googletagmanager.com/ns.html?id=GTM-PPFCWPB"height="0" width="0" style="display:none;visibility:hidden"></iframe>`